src/services/api/v2/contact.service.ts

```diff
@@ -0,0 +1,18 @@
+import Http, { HttpResponse } from "@/services/http.service";
+import ApiResource from "@/services/api/resource.service";
+import { ContactRequest, IncorrectInfoContactRequest } from '@/store/modules/contact'
+
+export default class Contact extends ApiResource {
+  baseVersion = "v2";
+  basePath = "feedback";
+
+  async postFeedback(feedback: ContactRequest|IncorrectInfoContactRequest): Promise<HttpResponse<ContactRequest|IncorrectInfoContactRequest>> {
+    return await Http.post<ContactRequest|IncorrectInfoContactRequest>(
+      this.formatEndpointUrl(""),
+      feedback,
+      {"Content-Type": "application/x-www-form-urlencoded"}
```

Comment:
Headers can be passed in via an Axios config. See change [here](https://github.com/petridishdev/orgbook-bc-client-vue/commit/1d8cbf87599b7b5b998a1fcaf2c013513cbde890#diff-78015307698270c08ff7c7830170acd5847318269e924388169144384195d84d) to Http service

Reply:
passed headers via axios config